assets/src/blocks/HubspotForm/hooks/useBackgroundImage.js

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from '@wordpress/element';
+
+export const useBackgroundImage = (image) => {
+  const [ backgroundImage, setBackgroundImage ] = useState();
+
+  useEffect(() => {
+    setBackgroundImage({ backgroundImage: image ? `url(${image})` : 'none' });
+  }, [ image ]);
+
+  return [
```

Comment:
Why do we return an array here? 🤔 It seems it would be simpler to just return the value